tsconfig.json

```
@@ -8,7 +8,8 @@
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"sourceMap": true,
-		"strict": true
+		"strict": true,
+		"ignoreDeprecations": "5.0"
```

Why is `ignoreDeprecations` set?

---

it is a temporary fix for the error below:
"Option 'importsNotUsedAsValues' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
  Use 'verbatimModuleSyntax' instead.ts"

---

What's the long-term fix?

---

'importsNotUsedAsValues' set to true. I will update this